increae the number of max errors to something more reasonable

--- webkitgtk-2.52.0/Source/cmake/WebKitCompilerFlags.cmake.old
+++ webkitgtk-2.52.0/Source/cmake/WebKitCompilerFlags.cmake
@@ -235,7 +235,7 @@
 
         # Match Clang's behavor and exit after emitting 20 errors.
         # https://bugs.webkit.org/show_bug.cgi?id=244621
-        WEBKIT_PREPEND_GLOBAL_COMPILER_FLAGS(-fmax-errors=20)
+        WEBKIT_PREPEND_GLOBAL_COMPILER_FLAGS(-fmax-errors=500)
     endif ()
 
     # Force SSE2 fp on x86 builds.
